src/main/java/io/debezium/mcs/envelope/ConnectorTypesEnvelope.java

```diff
@@ -0,0 +1,112 @@
+package io.debezium.mcs.envelope;
```

Comment:
Do we do license headers in MCS in general, @lburgazzoli?

Reply:
Neither COS nor KAS projects has this yet. I will ask Heiko if we need it and ask to create related tasks for it or for also missing LICENSE files (only kas-ui project in bf2 has it yet).